src/datagen/main/java/mekanism/common/recipe/impl/OreProcessingRecipeProvider.java

```diff
@@ -197,6 +197,12 @@ private void addDynamicOreProcessingIngotRecipes(Consumer<IFinishedRecipe> consu
               shard.getItemStack(4)
         ).build(consumer, Mekanism.rl(basePath + "shard/from_ore"));
         // Slurry
+        // pure
+        FluidSlurryToSlurryRecipeBuilder.washing(
+                FluidStackIngredient.from(MekanismTags.Fluids.HYDROFLUORIC_ACID, 5),
+                SlurryStackIngredient.from(slurry.getCleanSlurry(), 10),
+                slurry.getPureSlurry().getStack(12)
```

Comment:
This seems odd that it goes from 10 clean slurry to 12 purified slurry, as purifying something (that has a volume) does not usually cause you to get more than you put in. With ore processing for how more ingots come from the block, we are making less of the block get wasted, but when you have a specific amount `10 mB` I don't see how purifying it can get you `12 mB`